.github/workflows/ci.yml

```
@@ -218,28 +221,37 @@ jobs:
           cd frontend && bun run build && cd ..
         fi
         
-        echo "Starting wails dev server..."
+        echo "Starting wails dev server in background..."
         wails dev > wails.log 2>&1 &
         WAILS_PID=$!
         echo $WAILS_PID > wails.pid
+        echo "Wails dev server started with PID: $WAILS_PID"
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)